src/styles/index.scss

```diff
@@ -1,3 +1,199 @@
+* {
+  box-sizing: border-box;
+}
+
 body {
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
   margin: 0;
+  font-family: Arial, sans-serif;
+  font-size: 30px;
+  display: flex;
+}
+
+.calendar__day {
+  background-color: #eee;
+  border: 1px solid #000;
+  justify-content: center;
+  align-items: center;
+  width: 100px;
+  height: 100px;
+  transition-duration: 0.5s;
+  display: flex;
+}
+
+.calendar {
+  flex-wrap: wrap;
+  gap: 1px;
+  width: 726px;
+  padding: 10px;
+  display: flex;
+}
```

Comment:
The `display: flex;` property should be placed before `flex-wrap: wrap;`, `gap: 1px;` and `width: 726px;` properties. This is a common practice to first define a flex container and then specify its properties.